{"uuid":"c9b4f687-dfe9-476f-aea0-afbc1fd31b40","historyId":"19cc40902665743d9d690d0afca99b01","fullName":"org.uicatlog.base.BaseAndroidTest.setUp","labels":[{"name":"package","value":"org.uicatlog.tests.E2EMyntraTest"},{"name":"testClass","value":"org.uicatlog.tests.E2EMyntraTest"},{"name":"testMethod","value":"setUp"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"UICatalogIOS"},{"name":"subSuite","value":"org.uicatlog.tests.E2EMyntraTest"},{"name":"host","value":"DeWaNsHus-MacBook-Air.local"},{"name":"thread","value":"1427@DeWaNsHus-MacBook-Air.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"AS_ID","value":"-1"}],"links":[],"name":"setUp","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'DeWaNsHus-MacBook-Air.local', ip: 'fe80:0:0:0:4ad:4686:733f:b02f%en0'\nBuild info: version: '4.35.0', revision: '1c58e5028b'\nSystem info: os.name: 'Mac OS X', os.arch: 'aarch64', os.version: '26.0', java.version: '21.0.8'\nDriver info: io.appium.java_client.android.AndroidDriver\nCommand: [null, newSession {capabilities=[Capabilities {appium:appActivity: com.myntra.android.activiti..., appium:appPackage: com.myntra.android, appium:automationName: UiAutomator2, appium:deviceName: Pixel 8 API 36, appium:newCommandTimeout: 3600, appium:platformVersion: 16.0, platformName: ANDROID}]}]\nCapabilities {appium:appActivity: com.myntra.android.activiti..., appium:appPackage: com.myntra.android, appium:automationName: UiAutomator2, appium:deviceName: Pixel 8 API 36, appium:newCommandTimeout: 3600, appium:platformVersion: 16.0, platformName: ANDROID}","trace":"org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'DeWaNsHus-MacBook-Air.local', ip: 'fe80:0:0:0:4ad:4686:733f:b02f%en0'\nBuild info: version: '4.35.0', revision: '1c58e5028b'\nSystem info: os.name: 'Mac OS X', os.arch: 'aarch64', os.version: '26.0', java.version: '21.0.8'\nDriver info: io.appium.java_client.android.AndroidDriver\nCommand: [null, newSession {capabilities=[Capabilities {appium:appActivity: com.myntra.android.activiti..., appium:appPackage: com.myntra.android, appium:automationName: UiAutomator2, appium:deviceName: Pixel 8 API 36, appium:newCommandTimeout: 3600, appium:platformVersion: 16.0, platformName: ANDROID}]}]\nCapabilities {appium:appActivity: com.myntra.android.activiti..., appium:appPackage: com.myntra.android, appium:automationName: UiAutomator2, appium:deviceName: Pixel 8 API 36, appium:newCommandTimeout: 3600, appium:platformVersion: 16.0, platformName: ANDROID}\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:575)\n\tat io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:308)\n\tat org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:174)\n\tat io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:98)\n\tat io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:109)\n\tat io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:108)\n\tat org.uicatlog.base.BaseAndroidTest.setUp(BaseAndroidTest.java:32)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:188)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\n\tat org.testng.TestRunner.run(TestRunner.java:616)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\nCaused by: java.io.UncheckedIOException: java.net.ConnectException\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)\n\tat io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)\n\tat io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)\n\tat org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)\n\tat java.base/java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:317)\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java)\n\tat --- Async.Stack.Trace --- (captured by IntelliJ IDEA debugger)\n\tat java.base/java.util.concurrent.FutureTask.<init>(FutureTask.java:151)\n\tat java.base/java.util.concurrent.AbstractExecutorService.newTaskFor(AbstractExecutorService.java:98)\n\tat java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:122)\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.executeAsync(JdkHttpClient.java:384)\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute(JdkHttpClient.java:413)\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:89)\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:75)\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:61)\n\tat io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:178)\n\tat io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:243)\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:557)\n\t... 29 more\nCaused by: java.net.ConnectException\n\tat java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:955)\n\tat java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:133)\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)\n\t... 48 more\nCaused by: java.net.ConnectException\n\tat java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1065)\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:227)\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:280)\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$2(PlainHttpConnection.java:238)\n\tat java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)\n\tat java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)\n\tat java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)\n\tat java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run$$$capture(CompletableFuture.java:1773)\n\tat java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java)\n\tat --- Async.Stack.Trace --- (captured by IntelliJ IDEA debugger)\n\tat java.base/java.util.concurrent.CompletableFuture$AsyncSupply.<init>(CompletableFuture.java:1754)\n\tat java.base/java.util.concurrent.CompletableFuture.completeAsync(CompletableFuture.java:2719)\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection$ConnectEvent.handle(PlainHttpConnection.java:152)\n\tat java.net.http/jdk.internal.net.http.HttpClientImpl$SelectorManager.handleEvent(HttpClientImpl.java:1477)\n\tat java.net.http/jdk.internal.net.http.HttpClientImpl$SelectorManager.lambda$run$3(HttpClientImpl.java:1422)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\n\tat java.net.http/jdk.internal.net.http.HttpClientImpl$SelectorManager.run(HttpClientImpl.java:1422)\nCaused by: java.nio.channels.ClosedChannelException\n\tat java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:202)\n\tat java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:786)\n\tat java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:874)\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:210)\n\tat java.base/java.security.AccessController.doPrivileged(AccessController.java:571)\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:212)\n\t... 15 more\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1758872250012,"stop":1758872250013}